k8s/hyperconverged/tutorial-configure-openebs-local.md

```diff
@@ -56,22 +86,57 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
+snapshot-promoter    volumesnapshot.external-storage.k8s.io/snapshot-promoter   7m
 ```
 
 Some sample yaml files for stateful workoads using OpenEBS are provided in the [openebs/k8s/demo](https://github.com/openebs/openebs/tree/master/k8s/demo)
 
 ```
-ubuntu@kubemaster-01:~$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+ubuntu@kubemaster-01:~/openebs/k8s$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+deployment "jupyter-server" created
+persistentvolumeclaim "jupyter-data-vol-claim" created
+service "jupyter-service" created
 ```
 
 The above command will create the following, which can be verified using the corresponding kubectl commands:
 
 - Launch a Jupyter Server, with the specified notebook file from github
-  (kubectl get deployments)
+  ```
+  ubuntu@kubemaster-01:~/openebs/k8s$ kubectl get deployments
+  NAME                                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+  jupyter-server                                  1         1         1            0           2m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-ctrl   1         1         1            1           2m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-rep    3         3         3            1           2m
+  ```
 - Create an OpenEBS Volume and mounts to the Jupyter Server Pod (/mnt/data)
-  (kubectl get pvc)
-  (kubectl get pv)
-  (kubectl get pods)
+  ```
+  ubuntu@kubemaster-01:~/openebs/k8s$ kubectl get pvc
+  NAME                     STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+  jupyter-data-vol-claim   Bound     pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db   5G         RWO            openebs-jupyter   2m
+  
+  ubuntu@kubemaster-01:~/openebs/k8s$ kubectl get pv
+  NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                                   STORAGECLASS      REASON    AGE
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db   5G         RWO            Delete           Bound     default/jupyter-data-vol-claim           openebs-jupyter             2m
+  
+  ubuntu@kubemaster-01:~/openebs/k8s$ kubectl get pods
+  NAME                                                             READY     STATUS              RESTARTS   AGE
+  jupyter-server-68d4ddc48-tb2jj                                   1/1       Running             0          4m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-ctrl-7888fc7f89-q7blg   2/2       Running             0          4m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-rep-6d8d4965c6-9hrj7    1/1       Running             0          3m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-rep-6d8d4965c6-dj7sr    1/1       Running             0          3m
+  pvc-e3b4c8bc-7859-11e8-9472-02b983f0a4db-rep-6d8d4965c6-kkpt7    0/1       Pending             0          3m
+  ```
 - Expose the Jupyter Server to external world via the http://NodeIP:8888 (NodeIP is any of the minion nodes external IP)
```

Comment:
Expose = Display the Jupyter server by accessing the ...8888 URL (NodeIP is any minion nodes' external IP).